addons/website/static/src/js/content/website_root.js

```diff
@@ -35,6 +35,7 @@ export const WebsiteRoot = publicRootData.PublicRoot.extend({
      * @override
      */
     start: function () {
+        useBus(this.env.bus, "gmap_api_request", this._onGMapAPIRequest.bind(this));
```

Comment:
unless i misunderstand something, this will crash....

Reply:
you probably want to do something more explicit, adding and removing the listener on start/destroy.

Reply:
also, i am not sure that widgets have access to this.env